Common/UnitDefinitions/Capacitance.json

```diff
@@ -3,6 +3,7 @@
   "BaseUnit": "Farad",
   "XmlDocSummary": "Capacitance is the ability of a body to store an electric charge.",
   "XmlDocRemarks": "https://en.wikipedia.org/wiki/Capacitance",
+  "ObsoleteText": "This class is obsolete. Please use ElectricCapacitance instead",
```

Comment:
Minor, but I think it would be helpful if the text said it was renamed from X to Y so it is clear that only the name changed and they are equivalent.

Reply:
Does the ObsoleteText tag also parse XML comments? For example, would something like this work:
"<see cref="Capacitance" /> has been renamed to <see cref="ElectricCapacitance"> and marked as obsolete. Please change your code to use the new API before the next major revision of UnitsNet."

Reply:
> Does the ObsoleteText tag also parse XML comments? For example, would something like this work:
> 
> ```
> "<see cref="Capacitance" /> has been renamed to <see cref="ElectricCapacitance"> and marked as obsolete. Please change your code to use the new API before the next major revision of UnitsNet."
> ```

I don't think so- there is something about an [`UrlFormat`](https://learn.microsoft.com/en-us/dotnet/api/system.obsoleteattribute?view=net-9.0) property that could probably serve a similar purpose but I've never used it myself..

Reply:
> I don't think so- there is something about an [`UrlFormat`](https://learn.microsoft.com/en-us/dotnet/api/system.obsoleteattribute?view=net-9.0) property that could probably serve a similar purpose but I've never used it myself..

Yeah, that doesn't look like a good substitute. I'll confine the XML to the XmlDoc* tags for now, and will add it back if another method is found.